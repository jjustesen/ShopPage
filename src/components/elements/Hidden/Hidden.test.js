import React from "react";

import { render, cleanup } from "@sponte/lib-utils/dist/test-utils";

import { RstHidden } from "../..";

afterEach(cleanup);

describe("RstHidden", () => {
  it("deve renderizar e ser compativel com a versao anterior", () => {
    const { asFragment } = render(<RstHidden>Sponte</RstHidden>);
    expect(asFragment()).toMatchSnapshot();
  });
});
