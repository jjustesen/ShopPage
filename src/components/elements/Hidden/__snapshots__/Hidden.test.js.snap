// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RstHidden deve renderizar e ser compativel com a versao anterior 1`] = `
<DocumentFragment>
  <div
    class="sc-gzVnrw dPmAXc"
  >
    <div />
  </div>
  <div
    class="sc-dnqmqq sc-fjdhpX bcEbaQ"
  >
    Sponte
  </div>
</DocumentFragment>
`;
